src/prefect/orion/database/configurations.py

```diff
@@ -109,7 +109,12 @@ async def engine(self) -> AsyncEngine:
             if connect_args:
                 kwargs["connect_args"] = connect_args
 
-            engine = create_async_engine(self.connection_url, echo=self.echo, **kwargs)
+            engine = create_async_engine(
+                self.connection_url,
+                echo=self.echo,
+                execution_options={"isolation_level": "AUTOCOMMIT"},
```

Comment:
what are these changes related to? I'm not as familiar with our database configuration but my understanding is that changing the isolation level is a pretty significant change

Reply:
This was a weird merge with `main`. Should be gone now.